### PR TITLE
Separate  component  travel button

### PR DESCRIPTION
**Pull request by Amutha37**
This pull request addressing one of the request in putting from PR 68 to separate the button component .

- The new component "TravelMethodButton" will be rendered in "TravelMethodButttons" .
- 'TravelMethodIcon'  is moved into this new button component . It directly imported into this component .
-  The 'trello' for this task : https://trello.com/c/z50v3GrA/297-dev-travel-method-selection-page

- Please refer to this 'trello' link there is a question need attention to fix the way long text on the button wraps for mobile version. 

ADDITIONAL INFORMATION ****

1. PR68 and PR70 should be reviewed in that order .

2. PR68  should be reviewed first and merge.

3. PR70 to be reviewed and merged after PR68 .

### CODE REVIEW

File: components/TravelMethodButtons/TravelMethodButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                pos="absolute"
+                width={["30px", "32px"]}
+                height={["30px", "32px"]}
+                fontSize={"16px"}
+                mb={"22px"}
+              >
+                <Icon as={transportIcon[i]} />
+              </Box>
+              {mode}
+            </Button> */}
```
Comment (hqtan) [inline components/TravelMethodButtons/TravelMethodButtons.jsx:45]: Please remove the commented out code
```suggestion
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Almost there @Amutha37 !

I think this PR is ready to be merged once the commented out code has been removed
:tada: 

I think this PR can be merged as soon as the commented out code has been removed (unless @batteries76 , @OlaShabalina or @ymloh has more suggestions to make).

File: components/TravelMethodButtons/TravelMethodButtons.jsx (hunk 1 above)
Comment (Amutha37) [reply components/TravelMethodButtons/TravelMethodButtons.jsx]: This request has been addressed in the latest commit.

**Review by hqtan — APPROVED** [review]
lgtm

**Pull request merged**